dev/puppetserver.conf.sample

```diff
@@ -44,10 +44,15 @@ jruby-puppet: {
     # Puppet server expects to load Puppet from this location
     ruby-load-path: [./ruby/puppet/lib, ./ruby/facter/lib, ./ruby/hiera/lib]
 
-    # This setting determines where JRuby will look for gems.  It is also
-    # used by the `puppetserver gem` command line tool.
+    # This setting determines where JRuby will install gems.  It is used for loading gems,
+    # and also by the `puppetserver gem` command line tool.
     gem-home: ./target/jruby-gems
 
+    # This setting defines the complete "GEM_PATH" for jruby.  If set, it should include
+    # the gem-home directory as well as any other directories that gems can be loaded
+    # from (including the vendored gems directory for gems that ship with puppetserver)
+    gem-path: ./target/vendored-jruby-gems
```

Comment:
Based on the comment I would expect this value to be `./target/jruby-gems` and potentially others.